src/homework/day11/Runner1_2.java

```diff
@@ -0,0 +1,26 @@
+package homework.day11;
+
+//1.2
+//- создать 7 одновременно запускающихся потоков (например, в цикле for),
+// в каждом из которых, проитерировать коллекцию из 18 мышей, где каждая четная пикает
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Runner1_2 {
+    public static void main(String[] args) {
+        List<Mouse> listOfMouses = new ArrayList<>();
+        for (int mouseNumber = 1; mouseNumber <= 18; mouseNumber++) {
```

Comment:
Only FYI (not mandatory to fix) 
would be great to move this method to Utility class and re-use in all other classes,  because in each class you have this code